### PR TITLE
Make is2D a property, not a method

### PR DESCRIPTION
**Pull request by wilddamon**
Along the way, move  _computeMatrix and _generateCssString methods to be really private.

### CODE REVIEW

File: src/css-rotation.js [hunk 1]
```diff
@@ -76,17 +51,46 @@
...
+    }
+
+    for (var i = 0; i < arguments.length; i++) {
+      if (typeof arguments[i] != 'number') {
+        throw new TypeError('CSSRotation arguments must be of type \'number\'.');
+      }
+    }
+
+    this.is2D = (arguments.length == 1);
+    this.angle = this.is2D == 1 ? x : angle;
```
Comment (rjwright) [inline src/css-rotation.js]: Should just be `this.is2D ? x : angle;`

File: src/css-skew.js [hunk 2]
```diff
@@ -14,6 +14,20 @@
 
 (function(internal, scope) {
 
+  function computeMatrix(cssSkew) {
+    // CSSSkew represented by the 2D matrix:
+    //   1     tan(ax)  0
+    // tan(ay)    1     0
+
+    var tanAx = CSSSkew._tanDegrees(cssSkew.ax);
```
Comment (rjwright) [inline src/css-skew.js:22]: Shall we also take _tanDegrees out of CSSSkew?

**Comment by rjwright** [comment]
LGTM with nits

File: src/css-rotation.js (hunk 1 above)
Comment (wilddamon) [inline src/css-rotation.js]: Yes, done.

File: src/css-skew.js (hunk 2 above)
Comment (wilddamon) [inline src/css-skew.js:22]: That's actually a little more complicated than it looks due to the way that skew is tested. I'll do it in a separate PR.

**Pull request merged**